src/ext/java/org/opentripplanner/ext/digitransitemissions/DigitransitEmissionsModule.java

```diff
@@ -0,0 +1,115 @@
+package org.opentripplanner.ext.digitransitemissions;
+
+import com.csvreader.CsvReader;
+import dagger.Module;
+import jakarta.inject.Inject;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.zip.ZipFile;
+import org.opentripplanner.graph_builder.ConfiguredDataSource;
+import org.opentripplanner.graph_builder.GraphBuilderDataSources;
+import org.opentripplanner.graph_builder.model.GraphBuilderModule;
+import org.opentripplanner.gtfs.graphbuilder.GtfsFeedParameters;
+import org.opentripplanner.standalone.config.BuildConfig;
+import org.opentripplanner.transit.model.framework.FeedScopedId;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Module
+public class DigitransitEmissionsModule implements GraphBuilderModule {
```

Comment:
We shouldn't repeat Digitransit in class names as we have it in the package name. Although, perhaps emissions.digitransit.EmissionModule would be slightly more future proof package structure.

Reply:
Digitransit in names has been removed due to a comment by t2gran.